README.md

```diff
@@ -108,6 +108,14 @@ container registry where the image is stored, while the instrumented layers are
     '-Djdk.httpclient.HttpClient.log=requests,headers'
     ```
 
+## TypeSpec API Specifications
+
+    You can find the API specifications using (typespec)[https://github.com/microsoft/typespec] in typespec directory. Use following command to generate the API specifications.
+
+    ```bash
+    tsp compile typespec
+    ```
+    Check `tsp-output` directory for the generated API specifications.
```

Comment:
````suggestion
You can find the API specifications using (typespec)[https://github.com/microsoft/typespec] in typespec directory. Use following command to generate the API specifications.

```bash
tsp compile typespec
```

Check `tsp-output` directory for the generated API specifications.
````

Indentation here made the markdown render as a code block.